### ISSUE TITLE
Slow tldr -u (tldr --update)

### ISSUE THREAD

**Issue by osalbahr** [tldr-pages/tldr#9655]
## Expected behavior
tldr -u (tldr --update) should be quick if the local database is up to date.

## Actual behavior
tldr takes ~4s and sometimes more. I am not sure if it takes more with an empty local database, couldn't figure out how to wipe it.

## Log, debug output
```
$ time tldr -u
Successfully updated local database

real	0m4.761s
user	0m0.292s
sys	0m1.941s
$ time tldr -u
Successfully updated local database

real	0m4.146s
user	0m0.279s
sys	0m1.942s
$ time tldr -u
Successfully updated local database

real	0m3.851s
user	0m0.275s
sys	0m1.798s
```
## Environment
- Operating system - macOS/Linux/Windows (add version if needed)
macOS
- tldr version (`tldr --version`) 
```
$ tldr --version
tldr v1.5.0 (v1.5.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-c-client
```
- Installation
```
$ brew install tldr
Warning: tldr 1.5.0 is already installed and up-to-date.
To reinstall 1.5.0, run:
  brew reinstall tldr
```

(Template adapted from https://github.com/tldr-pages/tldr-node-client/issues)

**Comment by kbdharun** [comment]
Hi,
Thanks for the report. The client you are using with version [1.5.0](https://formulae.brew.sh/formula/tldr) is a [packaged](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tldr.rb) from our [C client](https://github.com/tldr-pages/tldr-c-client). So I will raise this issue over there and also in our chatroom.
